src/main/java/org/fundacionjala/coding/oscar/movies/Customer.java

```diff
@@ -0,0 +1,72 @@
+package org.fundacionjala.coding.oscar.movies;
+
+import java.util.Enumeration;
+import java.util.Vector;
+
+/**
+ * This class is for the customers.
+ */
+class Customer {
+    private String name;
+    private Vector rentals = new Vector();
+
+    /**
+     * This is the construct for customer.
+     *
+     * @param name of the customer.
+     */
+    Customer(String name) {
+        this.name = name;
+    }
+
+    /**
+     * tihs method add a rental to the vector.
+     *
+     * @param arg contained the movie with the days for rent.
+     */
+    public void addRental(Rental arg) {
+        rentals.addElement(arg);
+    }
+
+    /**
+     * This method return the name of the customer.
+     *
+     * @return string name
+     */
+    public String getName() {
+        return name;
+    }
+
+    /**
+     * This method has the logic for calculated the amount owed and the points earned.
+     *
+     * @return a string with all the datas.
+     */
+    public String statement() {
```

Comment:
This method is doing 3 things is calculating the totalAmount, totalFrequentRenterPoints and it is printing.